lang/src/org/partiql/lang/eval/physical/operators/AggregateOperatorFactory.kt

```diff
@@ -14,10 +14,27 @@
 package org.partiql.lang.eval.physical.operators
 
 import org.partiql.lang.domains.PartiqlPhysical
+import org.partiql.lang.eval.DEFAULT_COMPARATOR
+import org.partiql.lang.eval.ExprValue
+import org.partiql.lang.eval.physical.EvaluatorState
 import org.partiql.lang.eval.physical.SetVariableFunc
+import org.partiql.lang.eval.relation.RelationIterator
+import org.partiql.lang.eval.relation.RelationType
+import org.partiql.lang.eval.relation.relation
+import org.partiql.lang.planner.transforms.DEFAULT_IMPL_NAME
+import java.util.TreeMap
 
+/**
+ * Provides an implementation of the [PartiqlPhysical.Bexpr.Aggregate] operator.
+ *
+ * @constructor
+ *
+ * @param name
+ */
 public abstract class AggregateOperatorFactory(name: String) : RelationalOperatorFactory {
```

Comment:
```suggestion
abstract class AggregateOperatorFactory(name: String) : RelationalOperatorFactory {
```

Reply:
A couple more instances below but could you remove the public modifier?

Reply:
Explicit modifiers are used for Explict API Mode which we are moving towards https://kotlinlang.org/docs/whatsnew14.html#explicit-api-mode-for-library-authors